packages/core/src/enqueue_links/enqueue_links.ts

```diff
@@ -380,6 +385,19 @@ export async function enqueueLinks(
         }
     }
 
+    async function reportSkippedRequests(
+        skippedRequests: { url: string }[],
+        reason: Parameters<SkippedRequestCallback>[0]['reason'],
```

Comment:
nit: can we extract the union into a named type (`type SkippedRequestReason = 'filters' | ...`)?